src/codegen.py

```diff
@@ -59,6 +59,11 @@ def generate_source_file(template, peripheral, template_extension, output_dir):
         if _DEBUG:
             print(peripheral_data)
 
+        output_file_name = peripheral_data["info"]["title"]
+
+        # Naming convention: upper case
+        peripheral_data["info"]["title"] = peripheral_data["info"]["title"].upper()
```

Comment:
I don't think we'll need to change the filename here. If we change the filename `peripherals/Mcp4725.yaml` and in that file change `info.title` to be the uppercase form, that should be sufficient./

Reply:
Ok I see, so I will change the `info.title` in `Bmp280.yaml` and `Mcp4725.yaml` then. Thanks @Fleker !